src/bespokelabs/curator/prompter/prompter.py

```diff
@@ -50,6 +50,8 @@ def __init__(
         response_format: Optional[Type[BaseModel]] = None,
         batch: bool = False,
         batch_size: Optional[int] = None,
+        temperature: float = 1.0,
+        top_p: float = 1.0,
```

Comment:
Are these the right defaults to use?
Should it not be None?
https://docs.litellm.ai/docs/completion/input:
```
    temperature: Optional[float] = None,
```
If it is None, I assume litellm doesn't pass the temperature and the provider will the default that they have configured.

Reply:
Fixing this!